items/models.py

```diff
@@ -12,6 +12,21 @@ class Group(models.Model):
   title = models.CharField(max_length=150, default='Group Tasks')
   view = models.BooleanField(default=True)
   is_deleted = models.BooleanField(default=False)
+  RED = 'FF0000'
+  GREEN = '33CC00'
+  BLUE = '00CCFF'
+  ORANGE = 'FF9900'
+  WHITE = 'FFFFFF'
+  YELLOW = 'FFFF00'
```

Comment:
IMHO, it's not the coolest colours. Could you please try to use some colour generators? http://paletton.com or http://www.colorsontheweb.com/colorwizard.asp or anything else. 
Also it should be easy-expandable solution